lib/ansible/modules/network/aci/aci_firmware_policy.py

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/python
+
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: aci_firmware_policy
+
+short_description: This creates a firmware policy
+
+version_added: "2.8"
+
+description:
+    - This module creates a firmware policy for firmware groups. The firmware policy is create first and then
+    - referenced by the firmware group. You will assign the firmware and specify if you want to ignore the compatibility
+    - check
+options:
+    name:
+        description:
+            - Name of the firmware policy
+        required: true
+    version:
+        description:
+            - The version of the firmware assoicated with this policy. This value is very import as well as constructing
+            - it correctly. The syntax for this field is n9000-xx.x. If you look at the firmware repository using the UI
```

Comment:
This will not render correctly. If you use multiple dashes for the same sentence, they will end up being different paragraphs in the description. You don't need those subsequent dashes.